packages/chains/data/additional/59144.mjs

```diff
@@ -0,0 +1,32 @@
+// @ts-check
+/** @typedef { import("../../src/types").Chain } Chain */
+
+/** @type {Omit<Chain, "slug">} */
+export default {
+  name: "Linea Mainnet",
+  chain: "Linea Mainnet",
+  shortName: "linea-mainnet",
+  chainId: 59144,
+  testnet: false,
+  nativeCurrency: {
+    name: "Ether",
+    symbol: "ETH",
+    decimals: 18,
+  },
+  rpc: ["https://rpc.linea.build"],
```

Comment:
don't add this yet since it doesn't work (otherwise RPC edge will attempt to use this)